scripts/firebase-deploy

```diff
@@ -69,6 +72,28 @@ get_pr_num() {
 	echo "${pr_num}"
 }
 
+# update_links_in_file updates links in the given HTML file to subdomain,
+# e.g.: href="/docs" → href="https://docs.evy.dev" relevant subdomains are:
+# discord, docs, learn and play.
+update_links_in_file() {
+	local domain="$1" file="$2" target
+	target="firebase/${file#out/}"
+	sed -E -e 's href="/(discord|docs|learn|play) href="https://\1.'"${domain} g" "${file}" >"${target}"
+	diff -u "${file}" "${target}"
+}
+
+export -f update_links_in_file
+
+update_links() {
+	local environ="$1" domain="evystage.dev"
+	if [[ "${environ}" == "prod" ]]; then
+		domain="evy.dev"
+	fi
+
+	echo "update links $domain"
+	find out/public -name "*.html" -exec bash -c 'update_links_in_file "$0" "$1"' ${domain} {} \;
```

Comment:
Just a thought - feel free to ignore: I found this "$0" and "$1" a little weird, but the man page is reasonably explicit about extra args after the `-c` command string being set to $0 and onwards. I wonder if maybe you could pass `_` before `${domain}` so you can refer to the args as "$1" and "$2". There would still be a little weirdness but I think it would make more sense to someone casually reading it:

    find out/public -name "*.html" -exec bash -c 'update_links_in_file "$1" "$2"' _ ${domain} {} \;

Reply:
ok, I go with your feelings on bash - no idea how anybody else what feel/think about this. updated.